libstuff/libstuff.cpp

```diff
@@ -1318,10 +1319,9 @@ string SToJSON(const string& value, const bool forceString) {
     }
 
     // Otherwise, it's a string -- escape and return
-    // We need to escape all control characters in the string, not just the
-    // white-space control characters.
-    return "\"" + SEscape(value, "\x01\x02\x03\x04\x05\x06\x07\b\t\n\x0b\f\r\x0e\x0f\x10\x11"
-                                 "\x12\x13\x14\x15\x16\x17\x18\x19\x1a\x1b\x1c\x1d\x1e\x1f\x7f\"\\/",
+    // We need to escape all control characters in the string, not just the white-space control characters.
+    return "\"" + SEscape(value,     "\x01\x02\x03\x04\x05\x06\x07\x08\x09\x0a\x0b\x0c\x0d\x0e\x0f"
```

Comment:
```suggestion
    return "\"" + SEscape(value, "\x01\x02\x03\x04\x05\x06\x07\x08\x09\x0a\x0b\x0c\x0d\x0e\x0f"
```

Reply:
No need for the extra spacing, right?

Reply:
It lines up `1` over `1` so that it's easy to tell that both `x01-x0f` and `x11-x1f` are included in both lines and match.

Reply:
If you look at them in the file instead of in diff format it looks like:
```
    return "\"" + SEscape(value,     "\x01\x02\x03\x04\x05\x06\x07\x08\x09\x0a\x0b\x0c\x0d\x0e\x0f"
                                 "\x10\x11\x12\x13\x14\x15\x16\x17\x18\x19\x1a\x1b\x1c\x1d\x1e\x1f\x7f\"\\/",
```

Reply:
Ah then i have no strong opinion